Vesta/Marker/Planitiae.mkr

```diff
@@ -0,0 +1,10 @@
+BEGIN_HEADER
+InitialState on
+ColourIdx 2
+END_HEADER
+
+;Source: https://planetarynames.wr.usgs.gov/
+;28 November 2021
+
+BEGIN_DATA
+101.71	3.03	:	Feralia Planitia
```

Comment:
https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
This is not C coding. The extension .mkr designates this as a text file to be read by the simulator during launch to indicate labels to be applied to planetary features such as Mountain, or Crater. This is in the same format as the original author (mschweiger) wrote it as, this is updating the actual data. There should be no 'newline' needed, it wasn't included previously. The data reads longitude, latitude, and feature name, and label parameters. Thanks for the note, though. ARE

Reply:
This is not really about C (or any other language) coding. Handling of text files (especially on Linux where Orbiter might be ported one day) kind of expects that the content is correctly presented. Assuming text files are handled like stream of lines, very simple command like `cat *.mkr` will produce garbled output.

While not exactly "bad" it's not "good" either and this might/will cause issues.

"It's not about adding an extra newline at the end of a file, it's about not removing the newline that should be there."
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file